app/uk/gov/hmrc/fileupload/controllers/FileController.scala

```diff
@@ -25,18 +25,33 @@ import uk.gov.hmrc.fileupload.read.envelope.{Envelope, WithValidEnvelope}
 import uk.gov.hmrc.fileupload.read.file.Service._
 import uk.gov.hmrc.fileupload.write.envelope.{EnvelopeCommand, EnvelopeNotFoundError, FileNotFoundError, StoreFile}
 import uk.gov.hmrc.fileupload.write.infrastructure.{CommandAccepted, CommandNotAccepted}
-import uk.gov.hmrc.play.microservice.controller.BaseController
+import uk.gov.hmrc.fileupload._
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 
+
+package object file {
+
+  import play.api.libs.concurrent.Execution.Implicits._
```

Comment:
I think we should try to avoid that. Our application is doing different concerns (handles normal http endpoints, file streaming, reading and writing from db, ...) and therefore it could be necessary to use different execution context per concern. By using the generic play execution context we can not control the execution context for single concerns.